alphadia/workflow/manager.py

```diff
@@ -657,14 +677,27 @@ def load_classifier_store(self, path=None):
                         torch.load(os.path.join(path, file))
                     )
 
-    def get_classifier(self, available_columns):
+    def get_classifier(self, available_columns, version):
```

Comment:
please add type annotations for arguments and move the heading below `Parameters` :)
Should we also make `version` a kwarg with default -1 ?